hippunfold/config/snakebids.yml

```diff
@@ -358,7 +365,13 @@ cifti_metric_types:
     - gyrification.dscalar
     - curvature.dscalar
 
-
+# params for detecting and removing outlier vertices
```

Comment:
would be nice to put these under a common key, e.g. outlier_opts.. 
when the config file gets rewritten to and output folder all the ordering gets lost and so having things grouped is a good practice.. (generally something I think we should add an issue for in v2.0)

Reply:
[003dbf3](https://github.com/khanlab/hippunfold/pull/328/commits/003dbf36773548c6ebfd3df490cbe6f0e868c2e6)